src/Extensions/Logging/ServiceInitializationEventSource.cs

```diff
@@ -58,12 +58,26 @@ public void LogHostFailed(string exception, string serviceType)
 
 		private ServiceInitializationEventSource() { }
 
+		/// <summary>
+		/// Log host build succeeded
+		/// </summary>
+		/// <param name="hostProcessId">Host process id</param>
+		/// <param name="serviceType">The service type</param>
+		/// <param name="message">The message to be logged</param>
+		/// <param name="args">Extra arguments</param>
 		[Event((int)EventSourcesEventIds.GenericHostBuildSucceeded, Level = EventLevel.Informational, Message = "{2}", Version = 1)]
-		private void LogHostBuildSucceeded(int hostProcessId, string serviceType, string message) =>
-			WriteEvent((int)EventSourcesEventIds.GenericHostBuildSucceeded, hostProcessId, serviceType, message);
+		public void LogHostBuildSucceeded(int hostProcessId, string serviceType, string message, params object[] args) =>
```

Comment:
I don't think EventSource will work with `params` and `object` type and also I think it's not needed here

Reply:
The same goes for second event